### PR TITLE
Add uploading to PyPI on tag

### PR DESCRIPTION
**Pull request by jklukas**
Created project on PyPI: https://pypi.org/project/simpleprophet/

Related bug where we discussed how to provision PyPI creds: https://bugzilla.mozilla.org/show_bug.cgi?id=1586763

### CODE REVIEW

File: simpleprophet/setup.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+# Arguments marked as "Required" below must be included for upload to PyPI.
+# Fields marked as "Optional" may be commented out.
+
+setup(
+    name='simpleprophet',
+    # use_scm_version={"root": "..", "relative_to": __file__},
+    # setup_requires=['setuptools_scm'],
```
Comment (jezdez) [inline simpleprophet/setup.py]: I thought this should use setuptools-scm?

Comment (jklukas) [reply simpleprophet/setup.py]: Thank you. I removed this for testing. Will revert.

File: simpleprophet/setup.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        # These classifiers are *not* checked by 'pip install'. See instead
+        # 'python_requires' below.
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+    ],
+
+    install_requires=['fbprophet>=0.5'],
```
Comment (acmiyaguchi) [inline simpleprophet/setup.py]: Does this section also need to reference the dependencies specified in requirements.txt?

**Review by acmiyaguchi — APPROVED** [review]
r+, with one comment.

File: simpleprophet/setup.py (hunk 2 above)
Comment (jklukas) [reply simpleprophet/setup.py]: Yes, will do.

**Pull request merged**